docs/key-value-storage.md

```diff
@@ -0,0 +1,95 @@
+Your Nimbella account has been provisioned with Key-Value Storage and Object Storage. In this document, we are going to cover Key-Value Storage.
+
+A Key-Value Storage acts as a database where you create keys and apply values to those keys.
+
+The `nim` command will show your current content.
+
+```
+# To show the list of key-values (may be empty initially)
+nim key-value list
+```
+
+This document divides into these parts:
+
+- Conceptual Overview
+- Using a Key-Value Storage inside actions using a Nimbella SDK
+  - The `nodejs` SDK
+  - The `php` SDK
+  - The `python` SDK
+- Common Redis commands
+
+### Conceptual Overview
+
+A Key-Value Storage behaves like a database where you can create keys, apply values to those keys, and then call on them to extract the data that you need.
+
+In this visual example, you can store user info, such as their City, State, and Country, and apply the correct values to them. This allows you to keep track of data and request the data when needed.
+
+![](https://live.staticflickr.com/65535/50653417258_e27306c044_k.jpg)
+
+### Using a Key-Value Storage inside actions using a Nimbella SDK
+
+Nimbella provides SDKs for use within actions. Currently, these exist for `nodejs`, `php`, and `python`. An SDK is under development for `java` and the plan is to eventually cover all supported languages.
+
+### Using the `nodejs` SDK
```

Comment:
You'll want to convert these into language tabbed views. See this for an example https://github.com/nimbella/docs/pull/13.

Reply:
I tried to follow along with that. I pushed my changes and I'm hoping I did everything correctly.